packages/workflow/src/features/registration/fhir/fhir-bundle-modifier.ts

```diff
@@ -386,7 +386,8 @@ export function setupLastRegUser(
       valueReference: { reference: getPractitionerRef(practitioner) }
     })
   }
-  taskResource.lastModified = new Date().toISOString()
+  taskResource.lastModified =
```

Comment:
**This is the most important change in this PR**

Previously the timestamp always got generated for the first task in task history, which makes sense as we probably do not want anyone to submit it from outside. This now receives the timestamp I create in packages/gateway/src/features/registration/fhir-builders.ts and stores that as the timestamp instead